docs/explanation/system-snaps/upower/install-upower.md

```diff
@@ -0,0 +1,35 @@
+(explanation-system-snaps-upower-install-upower)=
+# Install Upower
+
+
+The Upower snap is currently available from the Snap Store. At can be installed on any system that supports snaps but is only recommended on [Ubuntu Core](/index) at the moment.
+
+You can install the snap with the following command:
+```bash
+snap install upower
```

Comment:
I'd recommend to use --channel with the track matching the UC release if on UC

Reply:
Ah good point - i'll add this